### ISSUE TITLE
[RAT] Progress report should show question numbers on the tiles.

### ISSUE THREAD

**Issue by jamesfid** [sixthedge/opentbl#81]
@jcamou I can hook up the number if you get the styles in.

**Comment by jcamou** [comment]
Yep, that's all working on the if-at so I can just paste it in once the numbers are in

**Comment by jcamou** [comment]
@jamesfid let me know when the numbers are in and I'll get this styled.
